src/gt4py/backend/gtc_backend/dace/backend.py

```diff
@@ -129,7 +201,8 @@ def __init__(self):
 
     def generate_dace_args(self, gtir, sdfg):
         offset_dict: Dict[str, Tuple[int, int, int]] = {
-            k: (-v[0][0], -v[1][0], -v[2][0]) for k, v in compute_legacy_extents(gtir).items()
+            k: (-v[0][0], -v[1][0], -v[2][0])
+            for k, v in compute_legacy_extents(gtir, mask_inwards=True).items()
         }
```

Comment:
Let's merge https://github.com/GridTools/gt4py/pull/671 before this and resolve the merge conflict.